relationshiptypes/radon.relationships.monitoring/IsMonitoredBy/RelationshipType.tosca

```diff
@@ -0,0 +1,38 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
+
+relationship_types:
+  radon.relationships.IsMonitoredBy:
+    derived_from: tosca.relationships.ConnectsTo
+    metadata:
+      targetNamespace: "radon.relationships"
+      abstract: "false"
+      final: "false"
+    interfaces:
+      Configure:
+        type: tosca.interfaces.relationship.Configure
+        inputs:
+          monitored_function:
+            type: string
+            required: true
+            default: { get_property: [ SOURCE, name ] }
+          pushgateway_ip:
+            type: string
+            required: true
+            default: { get_attribute: [ TARGET, pushgateway_ip ] }
+          user_email:
+            type: string
+            required: true
+            default: { get_property: [ TARGET, user_email ] }
+          grafana_ip: 
+            type: string
+            required: true
+            default: { get_property: [ TARGET, grafana_ip ] }
+          service_account_file:
+            type: string
+            required: true
+            default: "/tmp/service_account.json"
```

Comment:
If this cannot be defined as input I would suggest getting this information from the TARGET or SOURCE node.

Reply:
There is an issue here. Neither SOURCE nor TARGET have this info as property or attribute.
This can be defined as input and it is defined as input but I cannot get the value with get_input in this file.
I posted about that on slack

Reply:
We should create an issue on xopera about that.